src/js/requestHandlers.js

```diff
@@ -230,6 +235,85 @@ sjrk.storyTelling.server.deleteStoryFromCouch = function (id, deleteStoryDataSou
     return promise;
 };
 
+sjrk.storyTelling.server.deleteStoryFiles = function (storyContent, serverConfig) {
+    var filesToDelete = [];
+
+    fluid.each(storyContent, function (block) {
+        var blockFileName = "";
+
+        if (block.blockType === "image") {
+            blockFileName = block.imageUrl;
+        } else if (block.blockType === "audio" || block.blockType === "video") {
+            blockFileName = block.mediaUrl;
+        }
+
+        if (sjrk.storyTelling.server.isValidMediaFilename(blockFileName)) {
```

Comment:
You're not passing `serverConfig` to `isValidMediaFilename`. Though, I think it's actually better to remove the `access` check from that function anyway. We have a race condition: there is no guarantee that the file will still exist when we try to move it, so we need to handle the case that the file is missing there anyway.